src/main/java/org/jitsi/jicofo/JitsiMeetConferenceImpl.java

```diff
@@ -661,6 +661,7 @@ protected void onMemberJoined(final XmppChatMember chatRoomMember)
             logger.info(
                     "Member "
                         + chatRoomMember.getContactAddress() + " joined.");
+            getFocusManager().getStatistics().totalParticipants.incrementAndGet();
```

Comment:
would it be better to pass the statistics instance in the constructor instead?

Reply:
I'm not sure.  On one hand I'd say yes, but in general I think Jicofo embraces the osgi approach in favor of passing a bunch of services in to the constructor, so this felt more consistent.  If you think it'd be better as a constructor param I'll change it, I don't feel too strongly about it.

Reply:
I don't like OSGi and those static globals, but it doesn't matter, so I guess it's fine(actually checked and there are bunch of similar usages there already). Maybe at least not make it public or have it moved to the FocusManager class? It would be weird to start seeing other classes using this one to get FocusManager and it encourages just that.

Reply:
Ah, yes that was a mistake that it was public.  Thanks.